### PR TITLE
Hyundai: Add Car Port for 2022 Kia Niro EV

### PR DESCRIPTION
**Pull request by sunnyhaibin**
**Checklist**
- [x] added to RELEASES, CARS
- [x] test route added to [test_routes.py](https://github.com/sunnyhaibin/openpilot-1/blob/94ddfc7163f2f0cb5dbc9eea7df69606e4db0ab8/selfdrive/test/test_routes.py#L92)
- [x] route with openpilot: `c011e480f2430f67|2021-10-28--11-09-58`

Thanks to community Niro EV owner Kolibabas.

### CODE REVIEW

**Comment by kolibabas** [comment]
Not sure why it would fail on Route Not Found. I have the route preserved and set to public.

c011e480f2430f67|2021-10-28--11-09-58 

https://useradmin.comma.ai/?onebox=c011e480f2430f67%7C2021-10-28--11-09-58

File: docs/CARS.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ Community Maintained Cars and Features
 | Kia       | Forte 2018-21                 | SCC + LKAS        | Stock            | 0mph               | 0mph         |
 | Kia       | K5 2021                       | SCC + LFA         | Stock            | 0mph               | 0mph         |
 | Kia       | Niro EV 2019-21               | SCC + LKAS        | Stock            | 0mph               | 0mph         |
+| Kia       | Niro EV 2022                  | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```
Comment (adeebshihadeh) [inline docs/CARS.md:123]:
```suggestion
| Kia       | Niro EV 2019-22               | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```

**Comment by adeebshihadeh** [comment]
It's unclear that this needs a new car port. The existing Niro EVs never fingerprint correctly with the FW we have, so I suspect they're having the same issue as this 2022 Niro (i.e. the esp fw isn't from a Niro).

**Comment by sunnyhaibin** [comment]
Closing this PR as a car port is not required for the 2022 Kia Niro EV. The issue is caused by an incorrect firmware `esp` from a different car that was added to `KIA_NIRO_EV` during a previous firmware dump. New PR can be found here: #23253.

**Pull request closed without merging**